src/technologies/n.json

```diff
@@ -682,6 +682,39 @@
     "requires": "Next.js",
     "website": "https://nextjs.org/docs/app"
   },
+  "Next.js SSR": {
+    "cats": [
+      12
+    ],
+    "description": "Next.js Server-Side Rendering.",
+    "icon": "Next.js.svg",
+    "implies": [
+      "Next.js"
+    ],
+    "js": {
+      ".__NEXT_DATA__.gip": "true",
+      ".__NEXT_DATA__.gssp": "true"
+    },
+    "requires": "Next.js",
+    "website": "https://nextjs.org/docs/pages/building-your-application/rendering/server-side-rendering"
+  },
+  "Next.js SSG": {
+    "cats": [
+      12
```

Comment:
```suggestion
      57
```

Reply:
I think we should also remove 57 from Next.js above

Reply:
I've done that, but based on https://github.com/HTTPArchive/wappalyzer/pull/6#issuecomment-1854628880 I've included them in App Router as they might be SSG